run-services.sh

```
@@ -2,57 +2,156 @@
+}
 
-  echo "Stopping $service..."
-  docker-compose -f docker-compose.apps.yml stop "$service" || { echo "ERROR: FAILED TO STOP $service"; exit 1; }
+ACTION=""
+TARGET_SERVICES=()
+NO_BUILD=false
 
-  echo "Removing old container for $service..."
-  docker-compose -f docker-compose.apps.yml rm -f "$service" || { echo "ERROR: FAILED TO REMOVE $service CONTAINER"; exit 1; }
+run_compose() {
+  docker compose -f "$COMPOSE_FILE" $ACTIVE_PROFILES "$@"
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)